scripts/wait4.config.js

```diff
@@ -22,6 +22,13 @@ module.exports = {
           uri: 'localhost:3306',
           method: 'mysql',
           retries: 30
+        },
+        {
+          description: 'Auth Service',
+          /* Change host:port accordingly based on default.json attributbes. */
```

Comment:
wait4.config.js is Javascript file -> you can import default.json and pick up from it the host&port